src/Message/ImportCommandCreator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Message;
+
+use App\MessageHandler\BitbucketImportCommandHandler;
+use App\MessageHandler\GithubImportCommandHandler;
+
+class ImportCommandCreator {
+    public function create($username, $provider) {

```

Comment:
```suggestion
    public function create(string $username, string $provider): ImportCommand {
```

if we have abstract type for command.

Reply:
Fixed